app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/event_parser.rb

```diff
@@ -2,26 +2,26 @@ module ManageIQ::Providers::IbmCloud::VPC::CloudManager::EventParser
   def self.event_to_hash(event, ems_id)
     event_hash = {
       :event_type => event["action"],
-      :source     => "IBM_CLOUD_VPC",
+      :source     => "IBMCloud-VPC",
       :ems_id     => ems_id,
       :ems_ref    => event["_id"],
       :timestamp  => event["eventTime"],
       :full_data  => event
     }
 
     case event_hash[:event_type]
-    when 'is.instance.instance.create'
+    when /^is\.instance/
       parse_vm_event!(event, event_hash)
     end
 
     event_hash
   end
 
   def self.parse_vm_event!(event, event_hash)
-    return if event['responseData'].nil?
+    vm_ref = event.dig('target', 'id').match(/::instance:(?<instance_id>\S+)$/)
```

Comment:
Super minor and can be done in a follow-up, but you can do `.match()&.named_captures&.dig("instance_id")` that way you don't have `vm_ref.named_captures["instance_id"]` in multiple places below